internal/verifier/util.go

```diff
@@ -89,3 +91,26 @@ type QueryFilter struct {
 	Namespace string `bson:"namespace"    json:"namespace"`
 	To        string `bson:"to,omitempty" json:"to,omitempty"`
 }
+
+func (verifier *Verifier) doInMetaTransaction(
+	ctx context.Context,
+	todo func(context.Context, mongo.SessionContext) error,
```

Comment:
nit: I think `todo` is not the best name, given how we use `TODO` comments in our code base.